### PR TITLE
Add support for tongyi-thousand-questions (DashScope)

### PR DESCRIPTION
**Pull request by Cishoon**
## Description

This pull request adds support for Alibaba Tongyi Qianwen large model and embeddings. It includes the addition of the `dashscope` package.

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

This change has not yet been tested with new unit tests or updated existing tests.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] Made sure Checks passed

### CODE REVIEW

File: mem0/llms/qwen.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import json
+import os
+from typing import Dict, List, Optional
+
+from openai import OpenAI
+
+from mem0.llms.base import LLMBase
+from mem0.configs.llms.base import BaseLlmConfig
+
+class QwenLLM(LLMBase):
```
Comment (Panczo) [inline mem0/llms/qwen.py:10]: You could create a unit test to cover this class.

**Comment by Dev-Khant** [comment]
@Cishoon Can you please resolve the merge conflicts?

File: mem0/llms/qwen.py (hunk 1 above)
Comment (Cishoon) [reply mem0/llms/qwen.py:10]: Sorry for replying so late. I have create the unit test.

**Comment by Cishoon** [comment]
> @Cishoon Can you please resolve the merge conflicts?

Ok, I have resolved it.

**Comment by taranjeet** [comment]
Hey @Cishoon 
Thank you for your contribution! At the moment, we’re focusing on integrations that are highly requested by the community. We appreciate your effort and will keep this in mind for future priorities.

**Pull request closed without merging**